dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -1214,19 +1214,4 @@ public Response getToolZip(@ApiParam(hidden = true) @Parameter(hidden = true, na
         return Response.ok().entity((StreamingOutput) output -> writeStreamAsZip(sourceFiles, output, path))
             .header("Content-Disposition", "attachment; filename=\"" + fileName + "\"").build();
     }
-
-    @GET
```

Comment:
Probably ok to delete given that we're the only ones that really use this (and it isn't used a lot) 
But raising the idea of deprecating with an explicit "delete me" date